src/test/scala/edu/ie3/mobsim/io/geodata/PoiEnumsSpec.scala

```diff
@@ -10,9 +10,13 @@ import edu.ie3.mobsim.io.geodata.PoiEnums.{
   CategoricalLocationDictionary,
   PoiTypeDictionary
 }
+import edu.ie3.mobsim.model.{ElectricVehicleTestData, TripSimulationTestData}
 import edu.ie3.test.common.UnitSpec
 
-class PoiEnumsSpec extends UnitSpec {
+class PoiEnumsSpec
+    extends UnitSpec
+    with ElectricVehicleTestData
+    with TripSimulationTestData {
 
```

Comment:
What do you need these traits for here?

Reply:
There are unnecessary. I will remove them.